src/codec/ibc/core/client/v1/genesis.ts

```diff
@@ -420,7 +430,7 @@ var globalThis: any = (() => {
   if (typeof self !== 'undefined') return self;
   if (typeof window !== 'undefined') return window;
   if (typeof global !== 'undefined') return global;
-  throw new Error('Unable to locate global object');
+  throw 'Unable to locate global object';
```

Comment:
I wonder why this was changed. 😕

Reply:
Ask upstream. Another issue on ts-proto?